script.js

```diff
@@ -0,0 +1,432 @@
+$(document).ready(function () {
+
+    var checkIfSpell = true;
+    var classCheck = "";
+    //get all the inputs from the localstorage
+    if (localStorage.length != 0) {
+        //hit point local storage
+        if (localStorage.getItem("hitPoint") != "undefined") {
+            var hitPoint = localStorage.getItem("hitPoint", hitPoint);
+            $("#hp").val(hitPoint);
+
+        }
+        //armorclass local storage
+        if (localStorage.getItem("armorClass") != "undefined") {
+            var armorClass = localStorage.getItem("armorClass", armorClass);
+            $("#ac").val(armorClass);
+        }
+        //speed local storage
+        if (localStorage.getItem("speed") != "undefined") {
+            var speed = localStorage.getItem("speed", speed);
+            $("#sp").val(speed);
+        }
+        //initiative local storage
+        if (localStorage.getItem("initiative") != "undefined") {
+            var initiative = localStorage.getItem("initiative", initiative);
+            $("#int").val(initiative);
+        }
+        //profeciency local storage
+        if (localStorage.getItem("prof") != "undefined") {
+            var prof = localStorage.getItem("prof", prof);
+            $("#pb").val(prof);
+        }
+        //character local storage
+        if (localStorage.getItem("character") != "undefined") {
+            var character = localStorage.getItem("character", character);
+            $("#char-name").val(character);
+        }
+        //class local storage
+        if (localStorage.getItem("classbtn") != "Classarrow_drop_down") {
+            var classbtn = localStorage.getItem("classbtn", $('.classbtn').text());
```

Comment:
It's common to camelCase variables, so these buttons should be "classBtn", "levelBtn", "raceBtn", "backBtn"...